src/StorageFlattening.cpp

```diff
@@ -118,6 +118,11 @@ class FlattenDimensions : public IRMutator {
             shader_scope_realizations.pop(op->name);
         }
 
+        // The allocation bounds of the function taken into account of
+        // the align_storage directives. It is only used to determine the
+        // host allocation size and the strides in buffer_t objects (which
+        // also affects the device allocation in some backends).
+        vector<Expr> allocation_bounds(extents.size());
```

Comment:
allocation_bounds -> allocation_extents? Generally, "bounds" refers to both a min and an extent.